### DISCUSSION TITLE
IDS alert change

### DISCUSSION THREAD

**Discussion by kennysden** [Security-Onion-Solutions/securityonion#1971] [Unsupported Versions] [answered]
Are you able to switch from Zeek to Suricata with reinstallation?

**Comment by dougburks** [comment, marked as answer]
Zeek doesn't really generate IDS alerts the way that Suricata does. For more information about Zeek, please see:
https://docs.securityonion.net/en/2.3/zeek.html

**Reply by kennysden** [reply]
I accidentally hit Zeek during installation, is there a way that I can switch to Suricata without redoing the whole installation?

**Reply by dougburks** [reply]
If you are referring to the part of Setup where it asks you to choose between Zeek and Suricata, that is more to do with metadata logging rather than alerting. So if you chose Zeek there, you should still have Suricata generating IDS alerts.
